packages/contracts/scripts/deploy/dollar/solidityScripting/01_UbiquityDollarManager.s.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.16;
+
+import "./00_Constants.sol";
+
+contract ManagerScript is Constants {
+    UbiquityDollarManager manager; 
+    function run() public virtual {
+
+        uint256 deployerPrivateKey = vm.envUint("PRIVATE_KEY");
```

Comment:
they all inherits from constants where deployerPrivateKey is defined so we can remove it from all the run functions

Reply:
Thank you, I'd created a template and completely missed the redundancy.